src/helics/shared_api_library/backup/helics_api.h

```diff
@@ -9,151 +9,154 @@ SPDX-License-Identifier: BSD-3-Clause
 #include <stdlib.h>
 
 typedef enum {
-    HELICS_CORE_TYPE_DEFAULT = 0,
-    HELICS_CORE_TYPE_ZMQ = 1,
-    HELICS_CORE_TYPE_MPI = 2,
-    HELICS_CORE_TYPE_TEST = 3,
-    HELICS_CORE_TYPE_INTERPROCESS = 4,
-    HELICS_CORE_TYPE_IPC = 5,
-    HELICS_CORE_TYPE_TCP = 6,
-    HELICS_CORE_TYPE_UDP = 7,
-    HELICS_CORE_TYPE_ZMQ_SS = 10,
-    HELICS_CORE_TYPE_NNG = 9,
-    HELICS_CORE_TYPE_TCP_SS = 11,
-    HELICS_CORE_TYPE_HTTP = 12,
-    HELICS_CORE_TYPE_WEBSOCKET = 14,
-    HELICS_CORE_TYPE_INPROC = 18,
-    HELICS_CORE_TYPE_NULL = 66
+   HELICS_CORE_TYPE_DEFAULT = 0,
```

Comment:
Why is this 4 spaces changed to 3 spaces?

Reply:
this was fixed by formatting